src/MuTalk-Model/MutationTestingAnalysis.class.st

```diff
@@ -590,24 +656,48 @@ MutationTestingAnalysis >> stopOnErrorOrFail: aBoolean [
 	stopOnErrorOrFail := aBoolean
 ]
 
-{ #category : 'accesing' }
+{ #category : 'accessing' }
```

Comment:
haha even noticing typos in the protocols!

Reply:
I must admit, the warnings before committing helped me a bit ahah